src/api/localStorage.js

```diff
@@ -0,0 +1,93 @@
+export class LocalStorage {
+  constructor() {
+    this.users = JSON.parse(localStorage.getItem('userAccounts'));
+  }
+
+  createUser(user) {
+    if (this.users === null) this.users = [];
+    user._id = Math.floor(10000 + Math.random() * 9000);
+    this.users.push(user);
+    localStorage.setItem('UserAccounts',
```

Comment:
Maybe create a private `updateUsers` function to call when you need to update `localStorage` with the freshest `this.users`